src/Raven.Pal/src/posix/linuxonly.c

```diff
@@ -46,6 +48,7 @@ _sync_directory_allowed(int32_t dir_fd)
   case NFS_SUPER_MAGIC:
   case CIFS_MAGIC_NUMBER:
   case SMB_SUPER_MAGIC:
+  case SMB2_SUPER_MAGIC:
```

Comment:
you have defined `SMB2_MAGIC_NUMBER` but here we are using `SMB2_SUPER_MAGIC` ? Does it compile? Maybe if it does then `SMB2_SUPER_MAGIC` comes from somewhere else?

Reply:
😞 Sorry, PEBKAC. Thanks for finding it.